scripts/org-usage-report.sh

```diff
@@ -84,10 +84,17 @@ find_latest_login() {
   echo "$last_logon_times" | tr ' ' '\n' | sort | tail -n 1
 }
 
+quota_name() {
+  guid="$1"
+  quota_url=$(cf curl "/v2/organizations/${guid}" | jq -r '.entity.quota_definition_url')
```

Comment:
In `${org}` you still have the json object of the org, that would save one `curl` per org

```diff
diff --git a/scripts/org-usage-report.sh b/scripts/org-usage-report.sh
index 271d4ee2..11b5c14a 100755
--- a/scripts/org-usage-report.sh
+++ b/scripts/org-usage-report.sh
@@ -85,10 +85,8 @@ find_latest_login() {
 }
 
 quota_name() {
-  guid="$1"
-  quota_url=$(cf curl "/v2/organizations/${guid}" | jq -r '.entity.quota_definition_url')
-  name=$(cf curl "$quota_url" | jq -r '.entity.name')
-  echo "$name"
+  quota_url="$(jq -r '.entity.quota_definition_url')"
+  cf curl "$quota_url" | jq -r '.entity.name'
 }
 
 orgs=$(get_resources 100 "/v2/organizations")
@@ -123,7 +121,7 @@ for guid in $org_guids; do
   last_logon_time=$(find_latest_login "$users")
 
   # quota
-  quota=$(quota_name "$guid")
+  quota=$(echo "${org}" | quota_name "$guid")
 
   # Space-separated list of emails of org managers
   org_manager_emails=$(echo "$managers" | jq -r ".resources[].entity.username")
```